packages/framework/src/Model/Seo/Page/SeoPageDomain.php

```diff
@@ -23,87 +23,81 @@ class SeoPageDomain
      * @ORM\Id
      * @ORM\GeneratedValue(strategy="IDENTITY")
      */
-    private int $id;
+    protected $id;
 
     /**
      * @var int
      * @ORM\Column(name="domain_id", type="integer")
      */
-    private int $domainId;
+    protected $domainId;
 
     /**
      * @var string|null
      * @ORM\Column(type="text", nullable=true)
      */
-    private ?string $seoTitle;
+    protected $seoTitle;
 
     /**
      * @var string|null
      * @ORM\Column(type="text", nullable=true)
      */
-    private ?string $seoMetaDescription;
+    protected $seoMetaDescription;
 
     /**
      * @var string|null
      * @ORM\Column(type="text", nullable=true)
      */
-    private ?string $canonicalUrl;
+    protected $canonicalUrl;
 
     /**
      * @var string|null
      * @ORM\Column(type="text", nullable=true)
      */
-    private ?string $seoOgTitle;
+    protected $seoOgTitle;
 
     /**
      * @var string|null
      * @ORM\Column(type="text", nullable=true)
      */
-    private ?string $seoOgDescription;
+    protected $seoOgDescription;
 
     /**
-     * @var \App\Model\SeoPage\SeoPage
-     * @ORM\ManyToOne(targetEntity="App\Model\SeoPage\SeoPage", inversedBy="domains")
+     * @var \Shopsys\FrameworkBundle\Model\Seo\Page\SeoPage
+     * @ORM\ManyToOne(targetEntity="Shopsys\FrameworkBundle\Model\Seo\Page\SeoPage", inversedBy="domains")
      * @ORM\JoinColumn(name="seo_page_id", nullable=false, referencedColumnName="id", onDelete="CASCADE")
      */
-    private SeoPage $seoPage;
+    protected SeoPage $seoPage;
```

Comment:
it does not make sense to remove the typehint here (as well as the `Collection` typehint on the opposite side), right?

Reply:
I'm not sure... but in the sake of unification, I have removed the type hints in 8fc6688d9e